randovania/games/samus_returns/logic_database/Area 2 - Dam Interior.txt

```diff
@@ -415,7 +405,7 @@ Extra - asset_id: collision_camera012
                           # Breaking the Bomb block
                           Spring Ball or Mid-Air Morph (Intermediate)
                           # Dealing with the Wall Fire
```

Comment:
Could you change this comment to say `Wallfire` instead of `Wall Fire`? More accurate